diff -rupN nnet-master/nnet/convnet/conv.pyx nnet-recipe/nnet/convnet/conv.pyx
--- nnet-master/nnet/convnet/conv.pyx	2015-04-18 16:29:09.000000000 -0400
+++ nnet-recipe/nnet/convnet/conv.pyx	2015-08-04 11:37:21.083206564 -0400
@@ -128,3 +128,6 @@ def bprop_conv_bc01(np.ndarray[DTYPE_t,
                                 imgs_grad[i, c_imgs, img_y, img_x] += filters[c_imgs, c_convout, fil_y, fil_x] * convout_grad_value
                                 filters_grad[c_imgs, c_convout, fil_y, fil_x] += imgs[i, c_imgs, img_y, img_x] * convout_grad_value
     filters_grad[...] /= n_imgs
+    
+def print_test():
+    print("test conv")
diff -rupN nnet-master/nnet/convnet/pool.pyx nnet-recipe/nnet/convnet/pool.pyx
--- nnet-master/nnet/convnet/pool.pyx	2015-04-18 16:29:09.000000000 -0400
+++ nnet-recipe/nnet/convnet/pool.pyx	2015-08-04 11:37:36.915968620 -0400
@@ -100,3 +100,6 @@ def bprop_pool_bc01(np.ndarray[DTYPE_t,
                     img_y = switches[i, c, y, x, 0]
                     img_x = switches[i, c, y, x, 1]
                     imgs_grad[i, c, img_y, img_x] = poolout_grad[i, c, y, x]
+                    
+def print_test():
+    print("test pool")
diff -rupN nnet-master/nnet/neuralnetwork.py nnet-recipe/nnet/neuralnetwork.py
--- nnet-master/nnet/neuralnetwork.py	2015-04-18 16:29:09.000000000 -0400
+++ nnet-recipe/nnet/neuralnetwork.py	2015-08-03 19:58:14.079731928 -0400
@@ -1,5 +1,5 @@
 import numpy as np
-import scipy as sp
+#import scipy as sp
 from .layers import ParamMixin
 from .helpers import one_hot, unhot
 
@@ -114,5 +114,5 @@ class NeuralNetwork:
                         return np.ravel(self.layers[l].param_grads()[p])
 
                     param_init = np.ravel(np.copy(param))
-                    err = sp.optimize.check_grad(fun, grad_fun, param_init)
-                    print('diff %.2e' % err)
+                    #err = sp.optimize.check_grad(fun, grad_fun, param_init)
+                    #print('diff %.2e' % err)
diff -rupN nnet-master/setup.py nnet-recipe/setup.py
--- nnet-master/setup.py	2015-04-18 16:29:09.000000000 -0400
+++ nnet-recipe/setup.py	2015-08-04 12:30:43.646900569 -0400
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 
 import os
-import numpy as np
+#import numpy as np
 from setuptools import setup, find_packages
-from Cython.Build import cythonize
+#from Cython.Build import cythonize
 
 
 def read(fname):
@@ -19,7 +19,7 @@ setup(
     license = 'MIT',
     url = 'http://compute.dtu.dk/~abll',
     packages = find_packages(),
-    install_requires = ['numpy', 'scipy', 'cython'],
+    install_requires = ['numpy'], ## 'scipy'], ##'cython'],
     long_description = read('README.md'),
     classifiers = [
         'Development Status :: 4 - Beta',
@@ -30,8 +30,8 @@ setup(
         'Programming Language :: Python',
         'Topic :: Scientific/Engineering',
     ],
-    ext_modules = cythonize(['nnet/convnet/conv.pyx',
-                             'nnet/convnet/pool.pyx']),
-    include_dirs = [np.get_include()]
+    #ext_modules = cythonize(['nnet/convnet/conv.pyx',
+    #                         'nnet/convnet/pool.pyx']),
+    #include_dirs = [np.get_include()]
 )
 
