--- originalmodule/module/setup.py	2015-09-16 09:47:12.975557597 -0400
+++ awesome-cnn/module/setup.py	2015-09-16 09:26:12.453049687 -0400
@@ -1,25 +1,26 @@
 #!/usr/bin/env python
 
 import os
-import numpy as np
-from setuptools import setup, find_packages
-from Cython.Build import cythonize
+#import numpy as np
+#from setuptools import setup , find_packages
+from distutils.core import setup
+#from Cython.Build import cythonize
 
 
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
-print(find_packages())
+#print(find_packages())
 setup(
-    name = 'awesomecnn',
-    version = '0.18',
+    name = 'awesomecnn',
+    version = '0.18',
     author = 'Anders Boesen Lindbo Larsen',
     author_email = 'abll@dtu.dk',
     description = "Neural networks in NumPy/Cython",
     license = 'MIT',
     url = 'http://compute.dtu.dk/~abll',
-    packages = find_packages(),
-    install_requires = ['numpy', 'scipy', 'cython'],
+    packages = ['awesomecnn', 'awesomecnn.convnet'],# find_packages(),
+    install_requires = ['numpy', 'cython'], ## 'scipy'
     long_description = read('README.md'),
     classifiers = [
         'Development Status :: 4 - Beta',
@@ -29,10 +30,9 @@
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Topic :: Scientific/Engineering',
-    ],
-    ext_modules = cythonize(['awesomecnn/convnet/convx.pyx',
-                             'awesomecnn/convnet/poolx.pyx']),
-    include_dirs = [np.get_include()]
+    ]#,
+#    ext_modules = cythonize(['awesomecnn/convnet/convx.pyx',
+#                             'awesomecnn/convnet/poolx.pyx'])#,
+#    include_dirs = [np.get_include()]
 )
 
-
